### ISSUE TITLE
Provide feedback during and after image upload

### ISSUE THREAD

**Issue by aleene** [openfoodfacts/openfoodfacts-ios#647]
**Summary:** 
It would be nice to have more feedback on the image upload (both for existing and new products):
- a final image (thumbnail) when the upload has finished.
- a progress indicator during upload.

**Comment by aleene** [comment]
The refresh indicator could be replaced by a percentage upload indicator. After upload the image is already shown.

**Comment by aleene** [comment]
This should be part of the PictureViewModel

**Comment by aleene** [comment]
Advancing on this issue. I only have to replace the activity indicator with a progress indicator.

**Comment by aleene** [comment]
Note that this is dependent on issue #648

**Comment by aleene** [comment]
https://github.com/iwasrobbed/RPCircularProgress

**Comment by aleene** [comment]
There should also be indicators in each product section

**Comment by aleene** [comment]
![B02E7D79-5DCE-4093-AABF-5C0F35965850](https://user-images.githubusercontent.com/1329905/83918441-067da380-a779-11ea-9080-9b6beafa1b41.jpeg)

**Comment by teolemon** [comment]
Can we make it green ? Red looks stressful.

**Comment by aleene** [comment]
It turns green when it is ready
